src/app/static/src/tests/components/plots/table/table.test.ts

```diff
@@ -0,0 +1,247 @@
+import {createLocalVue, shallowMount} from "@vue/test-utils";
+import Table from "../../../../app/components/plots/table/Table.vue";
+import registerTranslations from "../../../../app/store/translations/registerTranslations";
+import Vuex from "vuex";
+import {emptyState} from "../../../../app/root";
+
+const localVue = createLocalVue();
+const store = new Vuex.Store({
+    state: emptyState()
+});
+registerTranslations(store);
+
+const propsData = {
+    selections: {
+        indicatorId: "prevalence",
+        detail: 4,
+        selectedFilterOptions: {
+            age: [{id: "0:15", label: "0-15"}],
+            sex: [{id: "female", label: "Female"}],
+            area: []
```

Comment:
Could we also have a test where an area filter is set to include one or more values, and check that this works with the detail level?